app/src/main/java/org/listenbrainz/android/di/ServiceModule.kt

```diff
@@ -35,7 +35,6 @@ class ServiceModule {
     
     
     @get:Provides
-    @get:Singleton
```

Comment:
Why would you not keep the services Singleton anymore?

Reply:
Will ping you on IRC for this one.